db/migrate/20191128003907_add_clearance_to_users.rb

```diff
@@ -0,0 +1,26 @@
+class AddClearanceToUsers < ActiveRecord::Migration[6.0]
+  def self.up
+    change_table :users do |t|
+      t.string :confirmation_token, limit: 128
+      t.string :remember_token, limit: 128
```

Comment:
Should these be nullable? If not, add a constraint. If you allow them to become `null` they will eventually (in some hair-pulling way) end up as `null`.